server/docs/rule-payload.json

```diff
@@ -1,13 +1,13 @@
 {
-    "id": "abc123",
-    "name": "Munich-weather-rule1-trigger-ads",
+    "name": "Hamburg-weather-rule1-trigger-ads",
+    "userId": "YrqYQc15jFYutbMdZNss",
```

Comment:
Can we take the user ID from the session? We should not accept the user defined "userId" value in the api.

Reply:
No user Id should be part of the payload, I thought about this, it will not be clear which endpoints do automatic user filtering and which not, i would also like to be able to call the api from a cli if needed in that case passing all the parameters makes it more useable.